generator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+echo "What do you want your project to be called?"
+
+read project_name
+
+mkdir $project_name
```

Comment:
@jeremywiebe what if, instead of doing that, let people know that we don't allow spaces / special characters, and then do something like `project_name=$(echo $project_name | tr -dc '[:alnum:]\n\r' | tr '[:upper:]' '[:lower:]')`?